CONTRIBUTING.md

```diff
@@ -165,7 +165,7 @@ All pull requests will undergo peer review. The review intends to ensure that a
 Participants in this project should review pull requests, which can be done using [GitHub's review interface](https://help.github.com/articles/about-pull-request-reviews/ "GitHub: about pull request reviews"). Following are the basic guidelines for reviewing pull requests:
```

Comment:
I agree that all results should also be pushed, if they are not in .gitignore.

I was wondering if you could clarify the case where you need to comment on codes that are not changed. Maybe add a paragraph as suggestion?

Reply:
Oh sorry I meant for Github when we are reviewing the PR. I couldn't find a place to comment on lines that are not changed. For example, I wanted to comment on lines for the staggered PR but since they are not changed here, I couldn't figure out how to add comments specific to those lines - and hence I am adding comments in a less relevant place. That's what I meant haha

Reply:
No worries. Thank you for the clarification @runjin326 . I think I get it.

I have not encountered cases like this in my work yet, so I am not sure how to best handle these cases. 

Therefore, I was wondering if you could draft a paragraph as suggestion for the best practice of handling such cases.

Reply:
Resolved with commit <https://github.com/PediatricOpenTargets/OpenPedCan-analysis/pull/73/commits/f4932a805de9f597523509569f8318860f6b6fc7>